server/service/src/sync/translations/vvm_status.rs

```diff
@@ -0,0 +1,95 @@
+use repository::{vvm_status_row::{VVMStatusRow, VVMStatusRowDelete}, StorageConnection, SyncBufferRow};
+
+use serde::Deserialize;
+
+use super::{PullTranslateResult, SyncTranslation};
+
+#[allow(non_snake_case)]
+#[derive(Deserialize)]
+pub struct LegacyVVMStatusRow {
+    ID: String,
+    description: String,
+    code: String,
+    level: i32,
+    is_active: bool,
+    unusable: bool,
+    option_id: String,
+}
+
+// Needs to be added to all_translators()
+pub(crate) fn boxed() -> Box<dyn SyncTranslation> {
+    Box::new(VVMStatusTranslation)
+}
+
+pub(super) struct VVMStatusTranslation;
+impl SyncTranslation for VVMStatusTranslation {
+    fn table_name(&self) -> &str {
+        "vaccine_vial_monitor_status"
+    }
+
+    fn pull_dependencies(&self) -> Vec<&str> {
+        vec![]
+    }
+
+    fn try_translate_from_upsert_sync_record(
+        &self,
+        _: &StorageConnection,
+        sync_record: &SyncBufferRow,
+    ) -> Result<PullTranslateResult, anyhow::Error> {
+        let data = serde_json::from_str::<LegacyVVMStatusRow>(&sync_record.data)?;
+        let result = VVMStatusRow {
+            id: data.ID,
+            description: data.description,
+            code: data.code,
+            level: data.level,
+            is_active: data.is_active,
+            unusable: data.unusable,
+            reason_id: Some(data.option_id),
```

Comment:
With the change above you'll need this...

```suggestion
            reason_id: data.option_id,
```